CHANGELOG.md

```diff
@@ -6,11 +6,75 @@ document.
 
 ## Unreleased / Beta / In Rust Nightly
 
-[66c29b97...master](https://github.com/rust-lang/rust-clippy/compare/66c29b97...master)
+[93f0a9a9...master](https://github.com/rust-lang/rust-clippy/compare/93f0a9a9...master)
+
+## Rust 1.78
+
+Current stable, released 2024-05-02
+
+[View all 112 merged pull requests](https://github.com/rust-lang/rust-clippy/pulls?q=merged%3A2024-01-26T05%3A46%3A23Z..2024-03-07T16%3A25%3A52Z+base%3Amaster)
+
+### Important Changes
+
+* In a few versions, `cargo-clippy` will now no longer ignore the first argument.
+  Passing in `clippy` as the first argument like cargo does will continue to work.
+  However, arguments like `--fix` will soon issue a warning and will no longer be.
+  `cargo clippy` (without `-`!) is unaffected by this change.
+  [#9461](https://github.com/rust-lang/rust-clippy/pull/9461)
+
```

Comment:
@flip1995 && @matthiaskrgr can you confirm that this part reflects the planned changes from #9461 correctly?

Reply:
> will soon issue a warning and will no longer be.

There's something missing at the end of this sentence.

I think what is missing here is, that arguments like `--fix` will only temporarily emit a warning but will change behavior (removing the warning again) 1-3 release cycles later.

Reply:
Also, once we merge the PR that actually introduces the warnings or changes the behavior we might want to write a blog post about it.

Reply:
Good catch. Now that I'm rereading this, why do we want to issue a warning for that? It feels weird that correct usage, like `cargo-clippy --fix` should issue a warning. Or was the idea just to issue a warning for unexpected arguments? Meaning that
`cargo-clippy --fix` would be accepted, but `cargo-clippy duck --fix` will issue a warning?

Reply:
Ping @flip1995 @matthiaskrgr, I've now updated this part to say the following

> * These changes will affect users of the `cargo-clippy` binary. Users of the 
>  `cargo clippy` command will be unaffected by the change.
>  
>    In a few versions, `cargo-clippy` will now no longer ignore the first argument.
>    Passing in `clippy` as the first argument like cargo does will continue to work.
>    There will be a transition period where unknown arguments will issue a warning
>    instead of a hard error.

Does that sound good?

Reply:
Almost. But it never will emit a hard error, it will change bahavior after the transition phase. This is also the answer to why we want to emit a warning first:

Currently `cargo-clippy --fix` does not run `--fix` but will do so in the future. So people using `cargo-clippy --fix` might want to remove the `--fix` flag before the behavior changes, so they can keep the current behavior. Otherwise they can ignore the warning. This warning should also be unaffected by `-Dwarnings`, so that it doesn't break CIs.